From 988a59b65bcf86ed1cdf9fefb5a2ac7c16536c1b Mon Sep 17 00:00:00 2001
From: Lennart Poettering <lennart@poettering.net>
Date: Tue, 27 Dec 2011 22:52:15 +0100
Subject: [PATCH] udev: exclude loopback device from udev rule based sysctl
 application, since we can just apply that directly at boot
 (cherry picked from commit
 330672957438243c8003d3a90f0e59dedbd845e9)

---
 src/99-systemd.rules.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/99-systemd.rules.in b/src/99-systemd.rules.in
index b2481ae..d306f71 100644
--- a/src/99-systemd.rules.in
+++ b/src/99-systemd.rules.in
@@ -44,7 +44,7 @@ SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", ENV{ID_USB_INTERFACES}=="*:0701??:
 
 # Apply sysctl variables to network devices (and only to those) as they appear.
 
-SUBSYSTEM=="net", RUN+="@rootlibexecdir@/systemd-sysctl --prefix=/proc/sys/net/ipv4/conf/$name --prefix=/proc/sys/net/ipv4/neigh/$name --prefix=/proc/sys/net/ipv6/conf/$name --prefix=/proc/sys/net/ipv6/neigh/$name"
+SUBSYSTEM=="net", KERNEL!="lo", RUN+="@rootlibexecdir@/systemd-sysctl --prefix=/proc/sys/net/ipv4/conf/$name --prefix=/proc/sys/net/ipv4/neigh/$name --prefix=/proc/sys/net/ipv6/conf/$name --prefix=/proc/sys/net/ipv6/neigh/$name"
 
 # Asynchronously mount file systems implemented by these modules as
 # soon as they are loaded.
-- 
1.7.7.5

